tck/settings.gradle.kts

```diff
@@ -1,4 +1,4 @@
-rootProject.name = "sparkplug-tck"
+rootProject.name = "tck"
```

Comment:
I don't think this is a good idea - this results in a jar file with the name tck-3.0.1, which isn't very informative. I think the longer name is necessary.

Reply:
In fact the eclipse guidelines probably expect it to be prefixed by "eclipse" as well but let's not change the name for now.

Reply:
the build from the IDE is not working correct with the name sparkplug-tck